[2024-06-14T09:57:02.185588+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","host":"localhost","port":null,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"pyroconnect","serverVersion":"8.0.32","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","host":"localhost","port":null,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"pyroconnect","serverVersion":"8.0.32","charset":"utf8mb4"}} []
[2024-06-14T09:57:02.187048+00:00] console.CRITICAL: Error thrown while running command "CheckInactiveFile". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/PyroConnect/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:80)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/PyroConnect/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at /var/www/html/PyroConnect/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"CheckInactiveFile","message":"An exception occurred in the driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)"} []
[2024-06-14T09:57:02.187267+00:00] console.DEBUG: Command "CheckInactiveFile" exited with code "1045" {"command":"CheckInactiveFile","code":1045} []
